sdk/playwrighttesting/microsoft-playwright-testing/src/utils/reporterUtils.ts

```diff
@@ -318,8 +318,10 @@ class ReporterUtils {
   };
 
   public static isTimeGreaterThanCurrentPlus10Minutes(isoString: string): boolean {
+    // Convert UTC to local
+    const localTimeString = new Date(isoString).toLocaleString();
     // Parse ISO 8601 string into a timestamp
```

Comment:
There's no guarantee it will be [ISO 8601](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleString) since it is by default dependent on customer's locale setting, and will produce different type of string formats.